tests/libvmi/operations.go

```diff
@@ -0,0 +1,31 @@
+package libvmi
+
+import (
+	. "github.com/onsi/gomega"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+)
+
+func CreateVmi(client kubecli.KubevirtClient, namespace string, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+	vmi, err := client.VirtualMachineInstance(namespace).Create(vmi)
+	if err != nil {
+		return nil, err
+	}
+	return vmi, nil
+}
+
+func DeleteVmi(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) error {
+	return virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &v12.DeleteOptions{})
+}
+
+func WaitForVmiDeletion(vmi *v1.VirtualMachineInstance) {
+	EventuallyWithOffset(1, matcher.ThisVMI(vmi), 120).Should(matcher.BeGone())
```

Comment:
Please do not use an assertion here, just return an error if the timeout is reached so the caller can handle it any way he wants. Lets keep assertions out of helpers as much as possible.

Reply:
Once we will decide on [this thread](https://github.com/kubevirt/kubevirt/pull/6331#discussion_r700997713) what should be done, I'll act accordingly. Originally I returned an error and did not assert but @rmohr asked me to assert.